### ISSUE TITLE
pursue better contrast on outlines for menus, dialogs, etc. especially in dark mode

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon#8785]
Especially in dark mode, the outlines of tooltips, menus, dialogs and other types of overlays can be under-pronounced.
In the below screenshot of a dialog, the low constrast between the gray box and the darker gray background results in an experience where things like the X close buttons seems to be floating in the middle of the page.
![image](https://user-images.githubusercontent.com/14298245/119692927-af953200-be00-11eb-95ea-4aee079a0547.png)

![image](https://user-images.githubusercontent.com/14298245/119890781-18f26f00-beed-11eb-9dd3-1e5b5fb3db40.png)

Although 'white space' can be used to separate the text of such non-persistent content from the background a demarcation of the border that achieves 3:1 would reduce potential failures of Non-text Contrast.

Note that this is more likely in the realm of 'ehancement' than 'bug' -- I don't think in most cases we'd fail [Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html), because one can argue that you don't need the border to identify the component. But the benefit of more contrast is clear and backed up by our research. This is especially useful for low vision users, who may have more difficulty locating this material when it appears, as well as more difficulty differentiating between this material and content behind it.

**Comment by mbgower** [comment]
One interesting thing I noticed was that in the [Overflow menu on the usage page](https://www.carbondesignsystem.com/components/overflow-menu/usage/), the focus indicator is used to create that clear boundary.
That's a little unusual, but may be interesting to explore as an overall solution.
![image](https://user-images.githubusercontent.com/14298245/119890946-45a68680-beed-11eb-931d-bc767283a1a0.png)

**Comment by aagonzales** [comment]
@mbgower That outline is what actually happens on all our dropdowns. I think a lot of our illustrated dropdowns in website images aren't showing the focus border but it does always appear in code for dropdowns. The focus stays on the whole menu then the hover style is what appears when arrowing through the options. It gets to be a lot of you have a focus outline around the menu and also a focus outline around the items. 

https://react.carbondesignsystem.com/?path=/story/components-dropdown--default
![image](https://user-images.githubusercontent.com/11670886/120014015-266c2f80-bfa7-11eb-890d-7566f9ba9e04.png)

**Comment by sstrubberg** [comment]
This might need to turn into a Github Discussion rather than an issue that needs to be addressed.
